### PR TITLE
Sample for voice guidance 

### PR DESCRIPTION
**Pull request by Merlin1stHere**
Here is an example that demonstrates voice skin management and voice guidance.

### CODE REVIEW

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        }
+
+        refreshVoicePackageList();
+    }
+
+    private void refreshVoicePackageList() {
+        m_packagesView.setAdapter(new VoicePackagesAdapter(this));
+    }
+
+    private class VoicePackagesAdapter
```
Comment (NazarKacharaba) [inline voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java]: make static if possible

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                }
+            });
+        }
+
+        @Override
+        public int getItemCount() {
+            return m_packages.size();
+        }
+
+        private class VoiceViewHolder extends RecyclerView.ViewHolder {
```
Comment (NazarKacharaba) [inline voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java]: make static if possible

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    private void refreshVoiceSkins() {
+        // Fill the list of locally downloaded voices.
+        VoiceCatalog catalog = VoiceCatalog.getInstance();
+        List<VoiceSkin> voiceSkins = catalog.getLocalVoiceSkins();
+
+        VoiceSkinsAdapter adapter = new VoiceSkinsAdapter(this, voiceSkins);
+        m_voiceSkinsView.setAdapter(adapter);
+    }
+
+    private class VoiceSkinsAdapter
```
Comment (NazarKacharaba) [inline voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java]: make static if possible

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                }
+            });
+        }
+
+        @Override
+        public int getItemCount() {
+            return m_voiceSkins.size();
+        }
+
+        private class VoiceViewHolder extends RecyclerView.ViewHolder {
```
Comment (NazarKacharaba) [inline voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java]: make static if possible

File: voice-navigation/app/build.gradle [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+repositories {
+    flatDir {
+        dirs 'libs'
+    }
+}
+
+dependencies {
+    implementation(name: 'HERE-sdk', ext: 'aar')
+    implementation 'com.google.code.gson:gson:2.8.2'
```
Comment (NazarKacharaba) [inline voice-navigation/app/build.gradle]: GSON library is not required for voice package + basic routing

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java (hunk 1 above)
Comment (Merlin1stHere) [reply voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java]: Done

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java (hunk 2 above)
Comment (Merlin1stHere) [reply voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoicePackagesActivity.java]: Done

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java (hunk 3 above)
Comment (Merlin1stHere) [reply voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java]: Done

File: voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java (hunk 4 above)
Comment (Merlin1stHere) [reply voice-navigation/app/src/main/java/com/here/android/example/voice/guidance/VoiceSkinsActivity.java]: Done

File: voice-navigation/app/build.gradle (hunk 5 above)
Comment (Merlin1stHere) [reply voice-navigation/app/build.gradle]: Done

**Pull request closed without merging**